README.md

```diff
@@ -117,6 +117,16 @@ jf 'rt u target/ my-repo/'
 
 > **_IMPORTANT:_** Notice the single quotes wrapping the command right after the **jf** step definition.
 
+If the JFrog CLI command contains some arguments with spaces, you can rather provide the arguments as a list:
+```groovy
+jf(['mvn', 'clean', 'install', '-Ddeploy.testProperty=Property with space'])
+```
+In this case, the quotes that would be required in a shell are not necessary (and would be part of the value itself, if used). 
+The above step is equivalent to a shell invocation as follows:
+![bash-spaces.png](images/readme/bash-spaces.png)
```

Comment:
I suggest using a console style text instead of an image. It looks better in my opinion.

Reply:
Notice - We agreed to keep it image, to make sure users do not copy the text